corehq/apps/sms/tests/inbound_handlers.py

```diff
@@ -24,6 +26,7 @@ def mock_critical_section_for_smsforms_sessions(contact_id):
     return MockContextManager()
 
 
+@flag_enabled('WEBAPPS_CASE_MIGRATION')
```

Comment:
what's this for?

Reply:
see https://github.com/dimagi/commcare-hq/pull/19449 for context, can remove this now that that PR is merged